walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -19,10 +19,27 @@
 
 public final class FindMeetingQuery {
 
+  private class AvailabilityChange {
+    long mandatoryAttendees = 0L;
+    long optionalAttendees = 0L;
+
+    public AvailabilityChange() {}
```

Comment:
You don't need to add the construtor explicitly if it's empty

Reply:
Not really, because I have another constructor, in this case I need to define default one as well.

https://stackoverflow.com/questions/3641114/do-i-really-need-to-define-default-constructor-in-java

Reply:
Sorry if I was not clear.

What I meant was: either make it non-empty by adding (`this(0L, 0L)`) to it or don't keep it and replace where it's called to pass `0L, 0L` explicitly.

An empty default constructor is easy to be unnoticed when we change the other constructors and forget to update it to match the behavior change. They're common to be used, for example, when you have a service that instantiates the class using reflection and requires the existence of an empty default constructor. In these cases, it's also a good practice to add an inline comment inside the empty constructor to explain why it exists

Reply:
I see, thanks)